Firestore/Example/App/CMakeLists.txt

```diff
@@ -16,35 +16,27 @@ if(NOT APPLE)
   return()
 endif()
 
-set(
-  sources
-  macOS/main.m
-  macOS/AppDelegate.m
-  macOS/AppDelegate.h
-)
-
-set(
-  resources
-  GoogleService-Info.plist
-)
-
 add_executable(
   firestore_example_app_macos MACOSX_BUNDLE
-  ${sources}
+  macOS/AppDelegate.h
+  macOS/AppDelegate.m
+  macOS/main.m
 )
 
-add_objc_flags(
+target_compile_options(
   firestore_example_app_macos
-  ${sources}
+  PRIVATE ${OBJC_FLAGS}
 )
 
 target_link_libraries(
   firestore_example_app_macos
-  PRIVATE "-framework AppKit"
+  PRIVATE
+  "-framework AppKit"
+  "-framework Foundation"
```

Comment:
How did it work before, without specifying the dependency on `Foundation`?

Reply:
It got the dependency implicitly because `AppKit` depends on `Foundation` too.